### PR TITLE
Fix docker image and hook up a github action to validate it

### PR DESCRIPTION
**Pull request by halkeye**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+#      - name: Lint Dockerfile
```
Comment (halkeye) [inline .github/workflows/docker.yml:12]: i commented this out as hadolint suggests that all the versions of apk packages are pinned. 
Options are:
* DIsable hadolint (did it)
* Pin Versions
* Put a hadolintignore in

**Comment by halkeye** [comment]
tests are failing because CC_TEST_REPORTER_ID  is undefined in PR

**Comment by halkeye** [comment]
https://github.com/ccarney16/pterodactyl-containers/blob/master/.github/workflows/update-panel.yml might be a better solution. I don't know github actions very well.

**Comment by renatolond** [comment]
Sorry, it took me a long time to come back to this. Do you mind rebasing your branch? I'll check the cc reporter issue :)

**Comment by halkeye** [comment]
:( looks like it was more than just resolving conflicts. Its been many months and since I have working upgraded version, coming back to this old PR is a little exhausting.

I recommend doing a compare with my fork and see what you want to keep.

**Pull request merged**